core/server/web/parent-app.js

```diff
@@ -38,9 +39,9 @@ module.exports = function setupParentApp(options = {}) {
     // API
     // @TODO: finish refactoring the API app
     // @TODO: decide what to do with these paths - config defaults? config overrides?
-    parentApp.use('/ghost/api/v0.1/', require('./api/v0.1/app')());
-    parentApp.use('/ghost/api/v2/content/', require('./api/v2/content/app')());
-    parentApp.use('/ghost/api/v2/admin/', require('./api/v2/admin/app')());
+    parentApp.use(urlUtils.getApiPath('deprecated'), require('./api/v0.1/app')());
+    parentApp.use(urlUtils.getApiPath('active'), require('./api/v2/content/app')());
+    parentApp.use(urlUtils.getApiPath('active', true), require('./api/v2/admin/app')());
```

Comment:
> urlUtils.getApiPath('active', true)

It's always hard to guess what "true" here means.
Let's use the option pattern?

`getApiPath({WHAT: active, admin: true})`

Reply:
👍 Yeah makes sense. I think might be better to have a separate PR which takes care of that as that will changes across the board along with tests, wdyt @kirrg001 ?

Reply:
Agree

Reply:
Actually the tests are not affected, only the method `urlFor` gets affected. 🤔